ffi/tests/read-table-testing/expected-data/basic-partitioned.expected

```diff
@@ -6,6 +6,14 @@ Schema:
 ├─ number: long
 └─ a_float: double
 
+letter:  [
```

Comment:
new way of doing it ends up putting it first, which is more correct based on the schema